### DISCUSSION TITLE
Not starting anymore after volume migration on Synology/Container Mng(Docker)

### DISCUSSION THREAD

**Discussion by rmmitev** [alexjustesen/speedtest-tracker#863] [Q&A]
Hello, i've installed the speedtest tracker container:latest and it was running for about a 1 month then i've upgraded my nas with a m2 drive and i've moved all my containers. The problem now is that it doesn't want to start. I've also tried fresh reinstall and completely reinstall my docker app incl. deleting all existing volumes but it didnt help....

I would be grateful if anyone could help.
Thanks in advance!

ps. My host is Synology with DSM 7.2-64570 Update 3. All other containers like pihole and unbound are running fine after the migration.
Log file attached.

[_speedtest-tracker_logs.txt](https://github.com/alexjustesen/speedtest-tracker/files/12862348/_speedtest-tracker_logs.txt)

**Comment by alexjustesen** [comment]
Check out #865
